pkg/rancher-desktop/components/Snapshots.vue

```diff
@@ -85,6 +88,9 @@ export default Vue.extend<Data, Methods, Computed, never>({
 
       return date.format('YYYY-MM-DD HH:mm');
     },
+    escapeSnapshotName(value: string|undefined) {
```

Comment:
This could also be:
```ts
escapeHtml,
```
That is, as a shorthand for
```ts
escapeHtml: escapeHtml
```
Assuming we change the interface to:
```ts
interface Methods {
  pollingStart: () => void,
  pollingStart: () => void,
  currentTime: () => string,
  currentTime: () => string,
  escapeHtml: (name: string|undefined) => string,
}
```
(That assume the method is named `escapeHtml`, instead of `escapeSnapshotName`; I feel like that makes more sense because it actually says what the method does.)